src/content/en/2021/performance.md

```diff
@@ -1,20 +1,534 @@
+  )
+}}
+
+FID performance by rank was flat.
+
+For all FID metrics, we see very large bars in the "good" category which makes it less effective unless we've truly hit peak performance. The good news is the <a hreflang="en" href="https://web.dev/better-responsiveness-metric/">Chrome team is evaluating this now</a> and would like your feedback.
+
+If your site's performance is not in the "good" category, then you definitely have a performance problem. A common culprit for FID issues is too much long-running JavaScript. Keep your bundle sizes small and pay attention to third-party scripts.
+
+### Total Blocking Time (TBT)
+
+> The Total Blocking Time (TBT) metric measures the total amount of time between First Contentful Paint (FCP) and Time to Interactive (TTI) where the main thread was blocked for long enough to prevent input responsiveness. --<a hreflang="en" href="https://web.dev/tbt/">Web.dev</a> ]
+
+<a hreflang="en" href="https://web.dev/tbt/">Total Blocking Time (TBT)</a> is a lab-based metric that helps us debug potential interactivity issues. FID is a field-based metric, and TBT is its lab-based analog. Currently, when evaluating client websites, I reach for total blocking time TBT as another indicator of possible performance issues due to JavaScript.
+
+Unfortunately, TBT is not measured in the Chrome User Experience Report. But, we can still get an idea of what's going on using the HTTP Archive Lighthouse data (only collected for mobile):
+
+{{ figure_markup(
+  image="performance-tbt.png",
+  caption="Lighthouse TBT scores",
+  description="Bar chart showing 31% of mobile pages with good TBT (<= 200ms), 11% with needs improvement, and 58% with poor (> 600ms) TBT.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=2135422883&format=interactive",
+  sheets_gid="730149797",
+  sql_file="tbt.sql"
+  )
+}}
+
+<p class="note"><strong>Note:</strong> The groups in the chart are based off of the Lighthouse score for TBT (e.g., >= 0.9 results in "good"). Due to rounding of the score, some TBT values slightly above 200ms get categorized as "good" (and similarly at the 600ms threshold).</p>
+
+Remember that the data is a single, throttled-CPU Lighthouse run through WebPageTest and does not reflect real user experiences. Yet, potential interactivity looked much worse when looking at TBT versus FID. The "real" evaluation of your interactivity is probably somewhere between. Thus, if your FID is "good", take a look at TBT in case you're missing some poor user experiences that FID can't catch yet. The same issues that cause poor FID also cause poor TBT.
+
+{{ figure_markup(
+  content="67 seconds",
+  caption="Longest TBT",
+  classes="medium-number",
+  sheets_gid="1423728540",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+## Conclusion
+
+We still have a long way to go to provide great user experience, but we can take steps to improve it.
```

Comment:
Seems a little negative, given that we've improved in last year according to the stats above (though appreciate some of this will be due to metric redefinition - but that's cause they needed redefining).

From the [Author's Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide):

> **The glass is half-full.** Sometimes we're drawn to shine a light on the fires burning in the tail of the web, but let's not forget about all the good stuff done by folks who are making active efforts to improve their UX. For example, we should highlight the [positive adoption rate of HTTPS](https://httparchive.org/reports/state-of-the-web#pctHttps) and not just dwell on the 20% of websites without it.

Reply:
I'm trying to balance that as I don't want people to take away that the job is done and they can stop worrying about it which is what I fear will happen if I only say it's positive. Does that make sense?

Reply:
Changed it to:

> Performance improved since 2020. Though we still have a long way to go to provide great user experience, we can take steps to improve it.